SpywareFilter/sections/tracking_servers.txt

```diff
@@ -42,6 +42,8 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+||reco.logger.co.kr^
```

Comment:
@piquark6046 
what do you think about `||logger.co.kr^$third-party`?

Reply:
Not bad.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/82419/commits/0b48eeccee6794a4142e1c5d52becd9266787bef